src/Orc.Controls/Controls/Callout/Models/CalloutManager.cs

```diff
@@ -0,0 +1,92 @@
+﻿namespace Orc.Controls
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Windows;
+    using System.Windows.Threading;
+    using Catel;
+    using Catel.Collections;
+    using Catel.MVVM;
+
+    public class CalloutManager : ICalloutManager
+    {
+        public CalloutManager()
+        {
+            _callouts = new List<IViewModel>();
+        }
+
+        private IList<IViewModel> _callouts;
+
+        public IList<IViewModel> Callouts
```

Comment:
It should be a list of ICallout objects. This way we can introduce new features *without breaking anything* in the future.